Maple2.File.Ingest/Mapper/MapEntityMapper.cs

```diff
@@ -81,22 +81,30 @@ public class MapEntityMapper : TypeMapper<MapEntity> {
                             };
                             continue;
                         case ISpawnPointNPC npcSpawn:
-                            int[] npcIds = npcSpawn.NpcList.Keys.TrySelect<string, int>(int.TryParse).ToArray();
-                            if (npcSpawn.NpcCount == 0 || npcIds.Length == 0) {
+                            IList<SpawnPointNPCListEntry> npcList = npcSpawn.NpcList.Select(entry => {
+                                if (!int.TryParse(entry.Key, out int npcId)) {
+                                    return null;
+                                }
```

Comment:
could we skip null entries instead? To ensure all entries are valid.

Reply:
OfType<> filters null values, I can change it to Where(x => x != null) if you think it'd be easier to read

Reply:
ohhh TIL. nice then that's fine